Video.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Date;
+
+public class Video {
+	private String title ;
+
+	private int priceCode ;
+	public static final int REGULAR = 1 ;
+	public static final int NEW_RELEASE =2 ;
+	
+	private int videoType ;
+	public static final int VHS = 1 ;
+	public static final int CD = 2 ;
+	public static final int DVD = 3 ;
+	
+	private Date registeredDate ;
+	private boolean rented ;

```

Comment:
Please refactoring Rantal information in Video.java file to Rental.java file.
I think move to Rental.java is natural.